src/SearchDlg.cpp

```diff
@@ -2423,6 +2443,93 @@ bool CSearchDlg::PreTranslateMessage(MSG* pMsg)
     return false;
 }
 
+LRESULT CSearchDlg::ColorizeMatchResultProc(LPNMLVCUSTOMDRAW lpLVCD)
+{
+    switch (lpLVCD->nmcd.dwDrawStage)
+    {
+        case CDDS_PREPAINT:                      // theme hooks this stage
+            return CDRF_NOTIFYITEMDRAW;
+        case CDDS_ITEMPREPAINT:
+            return CDRF_NOTIFYSUBITEMDRAW;
+        case  CDDS_ITEMPREPAINT | CDDS_SUBITEM:
+            return CDRF_NOTIFYPOSTPAINT | CDRF_NEWFONT;
+        case CDDS_ITEMPOSTPAINT | CDDS_SUBITEM: // use the theme color
+        {
+            if (IsDlgButtonChecked(*this, IDC_RESULTFILES) != BST_CHECKED && lpLVCD->iSubItem == 3)

```

Comment:
small performance issue: do the check `lpLVCD->iSubItem == 3` before `IsDlgButtonChecked(*this, IDC_RESULTFILES) != BST_CHECKED`

Reply:
Yeah, will be resolved.